test/eyeshade/publishers.integration.test.js

```diff
@@ -0,0 +1,124 @@
+'use strict'
+
+import { serial as test } from 'ava'
+import uuid from 'uuid'
+import {
+  cleanDbs,
+  cleanPgDb,
+  eyeshadeAgent,
+  connectToDb
+} from '../utils'
+import {
+  timeout
+} from 'bat-utils/lib/extras-utils'
+import { agent } from 'supertest'
+import Postgres from 'bat-utils/lib/runtime-postgres'
+
+const postgres = new Postgres({ postgres: { url: process.env.BAT_POSTGRES_URL } })
+
+test.afterEach(t => {
```

Comment:
should probably use `test.afterEach.always` to cover failures. also, probably best to use await inside of the fns.

Reply:
thanks, I resolved in https://github.com/brave-intl/bat-ledger/pull/523/commits/5079cc5e7dfb23bde3384bb6e7c1a19ed6982dc5